src/index.html

```diff
@@ -11,7 +11,177 @@
+            min="1"
+            max="100"
+            required
+            id="age"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            id="date"
+            name="birthDate"
+            required
+          >
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            id="agreement"
+            name="agreement"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-fieldset registration-info">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+            id="email"
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            id="password"
+            minlength="5"
+            maxlength="32"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-fieldset question-info">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="true"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="false"
+            >
+            No
+          </label>
+        </div>
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="favoriteColor"
+            id="color"
+          >
+        </label>
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="timeToSleep"
+            id="time"
+          >
+        </label>
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+            name="cars"
+            multiple
+            id="cars"
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="form-field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="range"
+            min="0"
+            max="10"
+            value="0"
+            id="range"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-fieldset additional-info">
+        <legend>Additional info</legend>
+        <label class="form-field">
+          Comments:
```

Comment:
checklist p.8 - Check the textarea behaviour - when you click on the textarea, the coursor should appear on the beginning of the input.

Reply:
not fixed 
<img width="362" alt="Screenshot 2023-02-20 at 11 50 57" src="https://user-images.githubusercontent.com/104769168/220076112-87f3e105-8990-4321-825e-af35ba459b16.png">